build.gradle

```diff
@@ -217,12 +218,6 @@ allprojects { currentProj ->
         ]
       }
 
-      // As of 2023-09-24, google-java-format cannot parse Java 21 language features.
```

Comment:
Should we retain this (updating it to "java22") on the theory that we may need it in the future?  Or is it better to just delete it and re-code it if needed?

Reply:
I would just recode as needed.  There's nothing tricky here.